<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="orders_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Менеджер заказов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="products_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Редактор продуктов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customers_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Клиентский центр</string>
    </property>
   </widget>
   <widget class="QPushButton" name="warehouses_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Инвентаризация по складам</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admins_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Nirmala UI Semilight&quot;;</string>
    </property>
    <property name="text">
     <string>Меню администратора</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
